### PR TITLE
Fix kbfs bin name on linux

### PR DESCRIPTION
**Pull request by gabriel**
@oconnor663

### CODE REVIEW

**Comment by gabriel** [comment]
Added the watchdog2 change (to exclude updater on linux)

File: go/install/install_linux.go [hunk 1]
```diff
@@ -66,3 +67,15 @@ func KBFSBinPath(runMode libkb.RunMode, binPath string) (string, error) {
...
+func kbfsBinName(runMode libkb.RunMode) (string, error) {
+	if runMode != libkb.ProductionRunMode {
+		return "", fmt.Errorf("KBFS install is currently only supported in production")
+	}
+	return "kbfsfuse", nil
+}
+
+func updaterBinName() (string, error) {
+	return "kbupdater", nil
+}
```
Comment (oconnor663) [inline go/install/install_linux.go]: I think this should return an error (or panic).

**Comment by oconnor663** [comment]
LGTM

**Comment by oconnor663** [comment]
LGTM with the one question above. The watchdog won't land in Linux packaging before tomorrow's rollout, but I'll get to it soon.

**Pull request merged**